### PR TITLE
using slap %!%

### PR DESCRIPTION
**Pull request by romainfrancois**
Just to illustrate that this is a perfect use case for `%!%` from my toy `slap` 📦

### CODE REVIEW

**Review by krlmlr — APPROVED** [review]
Nice idea, fine with me.

File: duckdb-rfuns-r/DESCRIPTION [hunk 1]
```diff
@@ -27,3 +28,4 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.1
 Remotes:
   duckdb/duckdb-r@v0.10.0
```
Comment (krlmlr) [inline duckdb-rfuns-r/DESCRIPTION:30]: Probably:

```suggestion
  duckdb/duckdb-r@v0.10.0,
```

Comment (krlmlr) [reply duckdb-rfuns-r/DESCRIPTION:30]: ...and, speaking of: what happens if we bump to v0.10.1? I guess I need to take a look...

Comment (romainfrancois) [reply duckdb-rfuns-r/DESCRIPTION:30]: in #74 the Remotes has disappeared as cran duckdb is now 0.10.1

**Comment by romainfrancois** [comment]
(I'll play with `slap` some more, and see if I get it to something that can be released before merging this. )

**Pull request merged**